### PR TITLE
Document how `.env` file needs to reside under `server/`

### PR DESCRIPTION
**Pull request by nick-funk**
## What does this PR do?

Document new `.env` file location in the docs for the mono-repo changes.

As doc says, the file needs to be placed at root of the `server/` sub-directory.

## These changes will impact:

- [ ] commenters
- [ ] moderators
- [ ] admins
- [X] developers

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indices.

## How do I test this PR?

- read doc change
- suggest rewording or different explanations if you think they are necessary

## Where any tests migrated to React Testing Library?

No

## How do we deploy this PR?

No special considerations.

### CODE REVIEW

**Review by kabeaty — COMMENTED** [review]
Nice! Can you please also update the mentions of the `.env` file in `environment-variables.md` and `installation.md`?

**Review by marcushaddon — APPROVED** [review]

**Review by kabeaty — APPROVED** [review]
